auditbeat/Makefile

```diff
@@ -1,10 +1,13 @@
 BEAT_NAME=auditbeat
 BEAT_TITLE=Auditbeat
 BEAT_DESCRIPTION=Audit the activities of users and processes on your system.
-SYSTEM_TESTS=false
-TEST_ENVIRONMENT=false
+SYSTEM_TESTS=true
+TEST_ENVIRONMENT=true
 GOX_OS?=linux windows ## @Building List of all OS to be supported by "make crosscompile".
 DEV_OS?=linux
+TESTING_ENVIRONMENT?=snapshot-noxpack
+GOPACKAGES=$(shell go list ${BEAT_PATH}/... | grep -v /vendor/)
```

Comment:
Do you know what requires this line? I'd assume this is the default in libbeat/scripts/Makefile.

Reply:
I just copied over from Metricbeat. It does not seem to be needed anymore. I also removed it from Metricbeat.